src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html

```diff
@@ -162,7 +162,9 @@ <h2 class="color-blue" *ngIf="showQuestion.length === 0 || showQuestion.includes
               Specify the feedback path that should be used to generate the appropriate feedback recipients
             </li>
           </ol>
-          <p>EXAMPLE BOX</p>
+          <tm-example-box>
+              <tm-question-edit-form [(formModel)]="exampleNumericalScaleEditFormModel" [isDisplayOnly]="true"></tm-question-edit-form>
```

Comment:
Let's just follow the convention in other PR by not subscribing to the change event (i.e. There is no need to have the `()` (the banana syntax)).

Reply:
Standardised!